### DISCUSSION TITLE
Add a way to print in A5 paper format

### DISCUSSION THREAD

**Discussion by Rilith** [emendelson/pdftoprinter#5] [Ideas]
Thank you for your work! Is it possible to implement an argoument for the command to print in a5? Thank you

**Comment by emendelson** [comment]
@Rilith - The only way I can think of is described in the "Customization" section on the page that describes this program. You will need to open PDFXChange, open the Print dialog, go the the printer's Page Setup menu, choose A5, and then save your settings as described on the description page.
